tests/test_runserver_serve.py

```diff
@@ -117,11 +117,12 @@ def __init__(self):
         self.on_response_prepare = []
         self.middlewares = []
         self.router = MagicMock()
+        self['static_root_url'] = '/static/'
 
 
 def test_modify_main_app_all_off(tmpworkdir):
     mktree(tmpworkdir, SIMPLE_APP)
-    config = Config(app_path='app.py', livereload=False, host='foobar.com')
+    config = Config(app_path='app.py', livereload=False, host='foobar.com', static_path='.')
```

Comment:
Afraid I haven't look at this code for a long time. Why are all these changes required?

Reply:
the default config objects have static_path=None
if static_path is None then the if block i added doesn't execute
then the app objects don't have app['static_root_url'] not set at all and the tests throw key errors

Maybe, it would be better, to add app['static_root_url'] to the app objects?

Reply:
the code change is fine, I just don't think you need to change this on every single test. Just change it on one test for full test coverage.

Reply:
the tests fail without it, because app object is missing 'static_root_url' key